app/javascript/toolbar-actions/util.spec.js

```diff
@@ -0,0 +1,43 @@
+import { convertMultParamsToRailsMultParams } from './util';
+
+const TESTCASES = [
+  [ // Example Case
+    "selectedUser=all&approvalStateCheckboxes=pending_approval&approvalStateCheckboxes=approved&approvalStateCheckboxes=denied&types=all&selectedPeriod=7&reasonText=",
+    "selectedUser=all&approvalStateCheckboxes[]=pending_approval&approvalStateCheckboxes[]=approved&approvalStateCheckboxes[]=denied&types=all&selectedPeriod=7&reasonText="
+  ], [
+    "foo=bar",
+    "foo=bar"
+  ], [
+    "foobar",
+    "foobar=undefined"
+  ], [
+    "foo=bar&tree&color=red",
```

Comment:
Another common case is to have a key with no value like

```
    "foo=&bar=",
```

Perhaps even an array where one of the values is blank?

```
    "foo=bar&foo=",
```